src/main/java/duke/helper/Parser.java

```diff
@@ -0,0 +1,82 @@
+package duke.helper;
+
+import duke.exception.EmptyTaskException;
+import duke.exception.InvalidCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+public class Parser {
+    /**
+     * Validates the task input, checks for empty tasks. If task is empty, throws EmptyTaskException.
```

Comment:
Good job in explaining when the exception will be thrown! Should the circumstance be explained on the same line as the comment with `@throws` instead?